doc/src/learn/architecture/consensus.md

```diff
@@ -1,20 +1,23 @@
 ---
-title: Narwhal and Tusk, Sui's Consensus Engine
+title: Narwhal, Bullshark, and Tusk, Sui's Consensus Engine
 ---
 
-This is a brief introduction to [Narwhal and Bullshark](https://github.com/MystenLabs/narwhal), the high-throughput mempool and consensus offered by Mysten Labs. Sui runs consensus as needed to periodically checkpoint its state. And for those transactions that require a total ordering, Narwhal/Bullshark is the consensus engine of Sui.
+This is a brief introduction to [Narwhal, Tusk](https://github.com/MystenLabs/narwhal), and [Bullshark](https://arxiv.org/abs/2201.05677), the high-throughput mempool and consensus offered by Mysten Labs. Sui runs consensus as needed to periodically checkpoint its state. And for those transactions that require a total ordering, Narwhal and either Bullshark or Tusk comprise the Sui Consensus Engine.
 
 The dual name highlights that the systems split the responsibilities of:
-- ensuring the availability of data submitted to consensus ([Narwhal](https://arxiv.org/abs/2105.11827))
-- agreeing on a specific ordering of this data ([Bullshark](https://arxiv.org/abs/2201.05677))
+- ensuring the availability of data submitted to consensus = [Narwhal](https://arxiv.org/abs/2105.11827)
+- agreeing on a specific ordering of this data = [Bullshark](https://arxiv.org/abs/2201.05677) or [Tusk ]
 
-> **Note:** Bullshark has replaced the Tusk component of the consensus protocol for reduced latency and support for fairness (where even slow validators can contribute). See [DAG Meets BFT - The Next Generation of BFT Consensus](https://decentralizedthoughts.github.io/2022-06-28-DAG-meets-BFT/) for a comparison of the protocols.
+In August 2022, Bullshark replaced the Tusk component of the consensus protocol as the default for reduced latency and support for fairness (where even slow validators can contribute). See [DAG Meets BFT - The Next Generation of BFT Consensus](https://decentralizedthoughts.github.io/2022-06-28-DAG-meets-BFT/) for a comparison of the protocols.
 
-This is done in two layered modules, so Narwhal can also be used coupled with an external consensus algorithm, such as HotStuff, Istanbul BFT, or Tendermint. Narwhal is undergoing integration in the [Celo](https://www.youtube.com/watch?v=Lwheo3jhAZM) and [Sommelier](https://www.prnewswire.com/news-releases/sommelier-partners-with-mysten-labs-to-make-the-cosmos-blockchain-the-fastest-on-the-planet-301381122.html) blockchain.
+Still, you may easily use Tusk instead of Bullshark by reverting the change shown in:
```

Comment:
may = have permission
vs
can = able to. I think this should be "you can easily"

Reply:
Actually, may makes sense here. Not only can they, we are actively endorsing that possibility:
https://www.merriam-webster.com/words-at-play/when-to-use-can-and-may